pkgs/development/libraries/freetype/default.nix

```diff
@@ -1,14 +1,24 @@
-{ lib, stdenv, fetchurl
-, buildPackages, pkgsHostHost
-, pkg-config, which, makeWrapper
-, zlib, bzip2, brotli, libpng, gnumake, glib
+{ lib
```

Comment:
The formatting change seems unrelated to the commit message.

Reply:
Please split this into two commits instead.

Reply:
Tis done!